proto/one/enterprise/report/report_run.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+import "google/protobuf/wrappers.proto";
+import "google/protobuf/field_mask.proto";
+import "record_auditinfo.proto";
+import "jsonTicksDateTime.proto";
+import "enum_rendering_engine.proto";
+
+option csharp_namespace ="ONE.Models.CSharp";
+
+message ReportRun{
+    
+    // Id of the ReportDefinition
+    string reportDefinitionid = 1; 
+    
+    // Name of Report Definition
+    google.protobuf.StringValue name = 2;
+    
+    // Report Definition owner
+    google.protobuf.StringValue ownerDigitalTwinRefId = 3;
+
+    //  Rendering Engine used 0:unknown 1:excel
+    EnumRenderingEngine renderingEngine = 4;
+    
+    // Id of the ReportDefinitionRun
+    string reportDefinitionRunId = 5;
+
+    // The name of the report template file
+    google.protobuf.StringValue sourceTemplateName = 6;
+    
+    // The name of the report output file
+    google.protobuf.StringValue reportOutputName = 7;
+    
+    // Start time of the report
+    JsonTicksDateTime startDate = 8;
+    
+    // End time of the report
+    JsonTicksDateTime endDate = 9;
+    
+    string createdById = 10;
```

Comment:
We have a RecordAuditInfo model that contains these two fields as well as modifiedById and modifiedOn.  Typically when using that we put it at the end of a model with the id 99 so if the model changes it is always at the end

Reply:
We don't have ModifiedOn  or ModifiedBy for the ReportDefinitionRun table so I didn't think I would be able to include RecordAuditInfo.

Reply:
You could still use it and just ignore those fields in mapping the protobuf to your database model, or you could add those columns to your sql table if desired, or you could leave it as is.

Reply:
Maybe I'll leave it for now and speak with Steve when he returns. I've sent him a message about the lack of the ModifiedBy/ModifiedOn fields so I just want to hear what his thoughts are

Reply:
@MatthewDeRose-AI maybe just create a tech debt item describing adding audit capability to the report run table and if it is deemed vital we bring it in later.